src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -444,9 +447,11 @@ class AgentCompletions extends Disposable {
 						const agentLabel = `@${agent.id}`;
 						const withSlash = `/${c.name}`;
 						return <CompletionItem>{
-							label: { label: withSlash, description: agentLabel },
+							label: withSlash,
```

Comment:
todo: only do this when in screen reader mode

Reply:
If it's not reading `label.description` but it is reading `description`, seems like those should be aligned?

Reply:
It seems like this is the behavior of the list. For example, in the editor's suggest list, the `description` is not read.

Reply:
https://github.com/microsoft/vscode/issues/167900

Reply:
I'll leave this with you and @jrieken to decide.

Reply:
It might make sense, especially in the suggest widget, to put all the extra detail behind some extra action. "press x for more detail". So they can hear the names briefly but it doesn't try to read a paragraph of text automatically

Reply:
we show the detail though by default, so should align this with the sighted experience. 

![Screenshot 2023-10-23 at 2 50 34 PM](https://github.com/microsoft/vscode/assets/29464607/ebbfacba-e87e-4515-aafc-77a8fbcbedf2)

Reply:
they can always press `Escape` to ignore it.